gammapy/spectrum/models.py

```diff
@@ -2,24 +2,90 @@
 """Spectral models for Gammapy.
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
+from ..extern.bunch import Bunch
+from . import CountsSpectrum
+import numpy as np
+
 
 __all__ = [
     'SpectralModel',
     'PowerLaw',
     'ExponentialCutoffPowerLaw',
 ]
 
+# Note: Consider to move stuff from _models_old.py here
 
 class SpectralModel(object):
     """Spectral model base class.
     """
+    def __call__(self, energy):
+        kwargs = self.parameters
+        kwargs.update(energy=energy)
+        return self.evaluate(**kwargs)
+
+    def predicted_counts(self, livetime, aeff, edisp):
```

Comment:
I agree that an observation class intuitively is the better place. However, the [SpectrumObservation](http://docs.gammapy.org/en/latest/api/gammapy.spectrum.SpectrumObservation.html) is a better place, since I already has accees to the 1D IRFs. The drwaback of putting the npred method here is, that people like @jjlk who only want to predict counts without having measuread ``n_on`` and ``n_off`` then cannot use this method. So I would say this is a show stopper. 
I will make a function 
```
def predicted_counts(model, arf, livetime, rmf)
```

and add a convenience method to the ``SpectrumObservation``, ok?

```